sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/lro/CancelHealthcareJob.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.lro;
+
+import com.azure.ai.textanalytics.TextAnalyticsClient;
+import com.azure.ai.textanalytics.TextAnalyticsClientBuilder;
+import com.azure.ai.textanalytics.models.HealthcareTaskResult;
+import com.azure.ai.textanalytics.models.TextAnalyticsOperationResult;
+import com.azure.ai.textanalytics.models.TextDocumentInput;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.util.Context;
+import com.azure.core.util.polling.PollResponse;
+import com.azure.core.util.polling.SyncPoller;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Sample demonstrates how to cancel a healthcare job.
+ */
+public class CancelHealthcareJob {
+    /**
+     * Main method to invoke this demo about how to cancel the healthcare long-running operation.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
+        TextAnalyticsClient client =
+            new TextAnalyticsClientBuilder()
+                .credential(new AzureKeyCredential("{key}"))
+                .endpoint("{endpoint}")
+                .buildClient();
+
+        List<TextDocumentInput> documents = new ArrayList<>();
+        for (int i = 0; i < 100; i++) {
```

Comment:
why the need for 100 documents?

Reply:
100 documents will take a long time to complete. This example focus on "Cancel the long-running operation".

Reply:
Not a big deal thought, just wondering if a small number will work too. (this could take time for parsing and encoding before sending it so it might make sample/test slow)

Reply:
changed to size of 10